### PR TITLE
feat: add manual generation of abi, runtime-capi

### PR DESCRIPTION
**Pull request by baszalmstra**
When using build.rs files several files get regenerated every build
which causes rebuilds even when nothing really changed. This commit
changes this behavior to make the user manually run a command when the
files should be regenerated.

### CODE REVIEW

**Comment by Wodann** [comment]
Is it necessary to do this for the CAPI too?

**Comment by baszalmstra** [comment]
> Is it necessary to do this for the CAPI too?

Appears to be the case yes. Without this cargo always detects changes.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -61,7 +61,9 @@ jobs:
 
       - name: Cargo check
         uses: actions-rs/cargo@v1
-        continue-on-error: ${{ matrix.config.toolchain != 'stable' }}
+        env:
```
Comment (Wodann) [inline .github/workflows/ci.yml]: Are you reverting this from your previous change?

File: crates/tools/src/abi.rs [hunk 2]
```diff
@@ -1,5 +1,9 @@
-use std::env;
-use std::path::PathBuf;
+use crate::{project_root, reformat, update, Result};
+use failure::format_err;
+use teraron::Mode;
+
+pub const RUNTIME_CAPI_DIR: &str = "crates/mun_runtime_capi";
```
Comment (Wodann) [inline crates/tools/src/abi.rs]:
```suggestion
```

This constant is not used in this file.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (baszalmstra) [reply .github/workflows/ci.yml]: Yes, I'm fixing this because the code should just build with Beta IMO

Comment (Wodann) [reply .github/workflows/ci.yml]: Okay. Maybe split this into a separate commit to clarify this (like you did for the previous PR)?

Comment (baszalmstra) [reply .github/workflows/ci.yml]: 👍 reverted

**Comment by Wodann** [comment]
Do you have any idea why the gen-runtime-capi check is failing just for ubuntu beta?

**Comment by baszalmstra** [comment]
> Do you have any idea why the gen-runtime-capi check is failing just for ubuntu beta?

No, I cant reproduce it either. But Im working on it. :)

**Comment by baszalmstra** [comment]
[Remove `runtime-ffi` submodule from `mun` repository](https://trello.com/c/0GtG3Vhn/98-remove-runtime-ffi-submodule-from-mun-repository)

**Pull request merged**